Formula/netcdf-mpi.rb

```diff
@@ -0,0 +1,47 @@
+class NetcdfMpi < Formula
+  desc "The NetCDF C I/O library compiled with MPI parallel features enabled"
+  homepage "https://www.unidata.ucar.edu/software/netcdf/"
+  url "https://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-c-4.7.4.tar.gz"
+  sha256 "0e476f00aeed95af8771ff2727b7a15b2de353fb7bb3074a0d340b55c2bd4ea8"
+  license "BSD-3-Clause"
+
+  depends_on "hdf5-mpi"
+
+  conflicts_with "netcdf", :because => "netcdf-mpi is an MPI parallel build, use either netcdf or netcdf-mpi"
+
+  def install
+    # move files that NetCDF wants to place in libdir,
+    # because brew does not want them there.
+    inreplace "./Makefile.in",
+          "settingsdir = $(libdir)",
+          "settingsdir = #{pkgshare}"
```

Comment:
If it's settings it should be `#{etc}` I think.

Reply:
OK, will make this change but btw, I got this solution from the `hdf5-mpi` formula.
https://github.com/Homebrew/homebrew-core/blob/f2ba900dbe72b43943b981aebcc0f2eb1760cc25/Formula/hdf5-mpi.rb#L33
perhaps that needs to be fixed as well.

Reply:
done.

Reply:
If these settings are meant to modified by the user then they should go in `etc`. If they are _not_ meant to be modified by the user, they should go in `pkgshare`.